scripts/tests/lit.spec.js

```diff
@@ -0,0 +1,40 @@
+const Helper = require('hubot-test-helper');
```

Comment:
Same thing with the location of the file here. Put it on the same level as `lit.js`.